resources/aws/instance.go

```diff
@@ -55,6 +55,16 @@ func statePendingOrRunning(instance *ec2.Instance) bool {
 	return false
 }
 
+func stateTerminated(instance *ec2.Instance) bool {
+	stateCode := *instance.State.Code
+	switch stateCode {
```

Comment:
Maybe we could introduce the switch when we have more codes to check, wouldn't work now `return *instance.State.Code == int64(EC2TerminatedState)` ?

Reply:
Yes, this was copy / paste from the existing check. This is much nicer, done.